server/routes.js

```diff
@@ -259,4 +271,22 @@ function fillDefaultSidebar(db) {
 				];
 				return context;
 			})
-}
\ No newline at end of file
+}
+
+function findFiles(db) {
+	return (context) =>
+		 fs.readdir( path.join(__dirname, "..", config.paths.upload) )
+		.then( (files) => {
+			context.files = files.map( (f) => {return {
+				path: path.join("/upload/", f),
+				delete: path.join("/static/", f), 
+				name: f.split("-").splice(1).join("-")
+			}})
+			context.files.unshift({
```

Comment:
So I'm not sure how you would prefer me to do it, assuming that I don't think it would be good to

 - Copy the templating source
 - Force a reload on upload
 - Do 100% client side templating

This seemed the best solution to me